### PR TITLE
explore page and etc. bugs and improvements

### PR DESCRIPTION
**Pull request by vanilla-james**

### CODE REVIEW

File: packages/beoble/components/Chat/ChatInput/ImageRows.tsx [hunk 1]
```diff
@@ -33,6 +33,18 @@ const StyledImage = styled(Image)<{ onClick?: () => void }>`
...
 
+const StyledImage = styled(Image)`
+  ${StyledMedia}
+`;
+
+const StyledVideo = styled.video`
+  ${StyledMedia}
+`;
+
+const isImgUrl = (url) => {
```
Comment (vanilla-james) [inline packages/beoble/components/Chat/ChatInput/ImageRows.tsx:44]: switch to a method that checks if
image resource's headers.get('Content-Type') is image or video

Comment (vanilla-james) [reply packages/beoble/components/Chat/ChatInput/ImageRows.tsx:44]: beoble dev server is returning 
```
has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```
so leaving as is for now

File: packages/beoble/components/DesktopView/Explore/Tabs/Web3CommunityChat/ChatroomSearchForm.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+              </>
+            );
+          }}
+        />
+      </SearchInputAndIconContainer>
+    </StyledSearchForm>
+  );
+}
+
+export default ChatroomSearchForm;
```
Comment (bamnenim-kryptonium) [inline packages/beoble/components/DesktopView/Explore/Tabs/Web3CommunityChat/ChatroomSearchForm.tsx:98]: use named export instead of default

**Review by bamnenim-kryptonium — APPROVED** [review]

**Comment by bamnenim-kryptonium** [comment]
LGTM, but you should wait for review from the issue reporter in [notion](https://www.notion.so/beoble/Explore-Page-Feedbacks-and-Issue-Report-8070a6f731ff48709aa9ea8107df1f87?pvs=4)

File: packages/beoble/components/DesktopView/Explore/Tabs/Web3CommunityChat/ChatroomSearchForm.tsx (hunk 2 above)
Comment (vanilla-james) [reply packages/beoble/components/DesktopView/Explore/Tabs/Web3CommunityChat/ChatroomSearchForm.tsx:98]: will fix

Comment (vanilla-james) [reply packages/beoble/components/DesktopView/Explore/Tabs/Web3CommunityChat/ChatroomSearchForm.tsx:98]: fixed in https://github.com/beoble/beoble-toolkit/pull/376/commits/8fe47b37b7cae62e9fa9d3614c96849c99b22791

**Pull request merged**